assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -399,17 +426,28 @@
       command "sed -i~orig -e 's/# deb-src/deb-src/' /etc/apt/sources.list"
     end
   when 22.04 # Ubuntu Jammy
+    package %w(krb5-multidev libkrb5-dev) do
+      action :install
+    end
     packages = general_packages.concat(debian_and_ubuntu_packages).concat(ubuntu_packages).concat(ubuntu_jammy_packages) - ['dh-systemd']
     execute 'enable apt sources' do
       command "sed -i~orig -e 's/# deb-src/deb-src/' /etc/apt/sources.list"
     end
+  when 24.04 # Ubuntu Noble
+    package %w(krb5-multidev libkrb5-dev) do
+      action :install
+    end
+    packages = general_packages.concat(debian_and_ubuntu_packages).concat(ubuntu_packages).concat(ubuntu_noble_packages) - ['dh-systemd', 'dpatch']
```

Comment:
Why modify list here, not in the packages list?

Reply:
These packages are common for all releases of Ubuntu. Therefore, the packages list can't be modified without breaking installation process for previous Ubuntu versions.

Reply:
Then remove them from all lists and put into corresponding lists for those versions that include this package.